PF/Assignment2/PascalTringle.java

```diff
@@ -0,0 +1,62 @@
+public class PascalTringle
+{
+
+	public static String spaces(int row, int n)
+	{
+		String spaceOutput = "";
+		int space;
+		if (row > n) {
```

Comment:
ACTION: What if row=100 and n=2? In that case it will return wrong string. Right?